docs/website/docs/user-guides/data-scientist.md

```diff
@@ -0,0 +1,156 @@
+a structured format. This enables them to access and analyze the data more effectively, improving
+their understanding of the underlying data structure.
+
+![structured-data](images/structured-data.png)
+
+A `dlt` pipeline is made of a source, which contains resources, and a connection to the destination,
+which we call pipeline. So in the simplest use case, you could pass your unstructured data to the
+`pipeline` and it will automatically be migrated to structured at the destination. See how to do
+that in our [pipeline documentation](../general-usage/pipeline).
+
+Besides strurdiness, this also adds convenience by automatically converting json types to db types,
+such as timestamps, etc.
+
+Read more about schema evolution in our blog:
+**[The structured data lake: How schema evolution enables the next generation of data platforms](https://dlthub.com/docs/blog/next-generation-data-platform).**
+
+### Data exploration
+
+Data Scientists require a comprehensive understanding of their data to derive meaningful insights
+and build accurate models. `dlt` can contribute to this by providing intuitive and user-friendly
+features for data exploration. It allows Data Scientists to quickly gain insights into their data by
+visualizing data summaries, statistics, and distributions. With `dlt`, data understanding becomes
+clearer and more accessible, enabling Data Scientists to make informed decisions throughout the
+analysis process.
+
+Besides, having a schema imposed on the data acts as a technical description of the data,
+accelerating the discovery process.
+
+See [Understanding the tables](../dlt-ecosystem/visualizations/understanding-the-tables),
+[Exploring the data](../dlt-ecosystem/visualizations/exploring-the-data) in our
+documentation.
+
+## Use case #3: Data Preprocessing and Transformation
+
+Data preparation is a crucial step in the data science workflow. `dlt` can facilitate data
+preprocessing and transformation tasks by providing a range of built-in features. It simplifies
+various tasks like data cleaning, anonymizing, handling missing values, data type conversion,
+feature scaling, and feature engineering. Data Scientists can leverage these capabilities to clean
+and transform their datasets efficiently, making them suitable for subsequent analysis and modeling.
+
+Python-first users can heavily customize how `dlt` sources produce data, as `dlt` supports
+selecting,
+[filtering](../general-usage/resource#filter-transform-and-pivot-data),
+[renaming](../general-usage/customising-pipelines/renaming_columns),
+[anonymizing](../general-usage/customising-pipelines/pseudonymizing_columns),
+and just about any custom operation.
+
+Compliance is also a case where preprocessing is the way to solve the issue: Besides being
+python-friendly, the ability to apply transformation logic before loading data, allows us to
+separate, filter or transform sensitive data.
```

Comment:
remove comma in "before loading data, allows us to separate" - alternatively, we need to add subject and verb for the preceding sentence ("dlt has the ability...")